### PR TITLE
Toby2

### PR DESCRIPTION
**Pull request by TitanSri**
Deepak please review my code for FixItemControl and FixItemUI

### CODE REVIEW

File: BRLS/src/library/payfine/PayFineControl.java [hunk 1]
```diff
@@ -26,9 +26,9 @@ public void setUI(PayFineUI ui) {
     }
```
Comment (TitanSri) [inline BRLS/src/library/payfine/PayFineControl.java:26]: testing to see if others can see my comment

File: BRLS/src/library/Main.java [hunk 2]
```diff
@@ -4,9 +4,9 @@
 import library.borrowitem.bORROW_IteM_cONTROL;
 import library.entities.*;
 import library.fixitem.FixItemUI;
-import library.fixitem.fIX_iTeM_cONTROL;
+import library.fixitem.FixItemControl;
```
Comment (DChand2021-2023) [inline BRLS/src/library/Main.java:7]: As per the class diagram, this the the correct class name.

Comment (TitanSri) [reply BRLS/src/library/Main.java:7]: i can see it

Comment (DChand2021-2023) [inline BRLS/src/library/Main.java:7]: # Class name correct as per class diagram and required convention.

**Review by DChand2021-2023 — COMMENTED** [review]
Review done for PayFineControl, PayFineUI, FixItemControl & FixItemUI. No defects found.

File: BRLS/src/library/fixitem/FixItemUI.java [hunk 3]
```diff
@@ -4,99 +4,82 @@
...
-	public void RuN() {
-		DiSpLaY_OuTpUt("Fix Item Use Case UI\n");
+    public void run() {
+        displayOutput("Fix Item Use Case UI\n");
 		
-		while (true) {
+        while (true) {
 			
-			switch (StAtE) {
+            switch (uiState) {
```
Comment (DChand2021-2023) [inline BRLS/src/library/fixitem/FixItemUI.java:25]: Defect 1 PR#2.
Indentation needs to be addressed for the nested switch statements as per rule 9 of the coding style guideline.

File: BRLS/src/library/payfine/PayFineUI.java [hunk 4]
```diff
@@ -5,117 +5,110 @@
...
-				}
-				try {
-					long PAtroN_ID = Long.valueOf(PaT_Str).longValue();
-					CoNtRoL.CaRd_sWiPeD(PAtroN_ID);
-				}
-				catch (NumberFormatException e) {
-					DiSpLaYoUtPuT("Invalid patronID");
-				}
-				break;
+            case READY:
```
Comment (DChand2021-2023) [inline BRLS/src/library/payfine/PayFineUI.java:31]: Defect 2 PR#2.
Indentation needs to be addressed for the nested switch statements as per rule 9 of the coding style guideline.

**Review by DChand2021-2023 — CHANGES_REQUESTED** [review]
Two defects found. Both are based on indentation of styling code guideline rule 9.

File: BRLS/src/library/fixitem/FixItemUI.java (hunk 3 above)
Comment (TitanSri) [reply BRLS/src/library/fixitem/FixItemUI.java:25]: I changed it, can you check

Comment (TitanSri) [reply BRLS/src/library/fixitem/FixItemUI.java:25]: fixed, please review again

File: BRLS/src/library/payfine/PayFineUI.java (hunk 4 above)
Comment (TitanSri) [reply BRLS/src/library/payfine/PayFineUI.java]: fixed, please review again

File: BRLS/src/library/fixitem/FixItemUI.java (hunk 3 above)
Comment (DChand2021-2023) [reply BRLS/src/library/fixitem/FixItemUI.java:25]: Looks good.

**Review by DChand2021-2023 — APPROVED** [review]

**Review by DChand2021-2023 — CHANGES_REQUESTED** [review]
Toby,  2 defects based on rule 9 was identified. Please address them.

File: BRLS/src/library/fixitem/FixItemControl.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
+package library.fixitem;
+import library.entities.Item;
+import library.entities.Library;
+
+public class FixItemControl {
+	
+    private enum ControlState {INITIALISED, READY, INSPECTING };
```
[the remark this answers is not in the document]
Comment (TitanSri) [reply BRLS/src/library/fixitem/FixItemControl.java:7]: Fixed please check

File: BRLS/src/library/fixitem/FixItemUI.java [hunk 6]
```diff
@@ -4,99 +4,82 @@
 
 public class FixItemUI {
 
-	private enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
-
-	private fIX_iTeM_cONTROL CoNtRoL;
-	private Scanner InPuT;
-	private uI_sTaTe StAtE;
+    private enum FixItemUIState {INITIALISED, READY, INSPECTING, COMPLETED };
```
[the remark this answers is not in the document]
Comment (TitanSri) [reply BRLS/src/library/fixitem/FixItemUI.java]: Fixed please check

File: BRLS/src/library/fixitem/FixItemControl.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (DChand2021-2023) [reply BRLS/src/library/fixitem/FixItemControl.java:7]: Can’t see the changes:
private enum ControlState {INITIALISED, READY, **INSPECTING };**

File: BRLS/src/library/fixitem/FixItemUI.java (hunk 6 above)
[the remark this answers is not in the document]
Comment (DChand2021-2023) [reply BRLS/src/library/fixitem/FixItemUI.java]: All done.

File: BRLS/src/library/payfine/PayFineUI.java (hunk 4 above)
Comment (DChand2021-2023) [reply BRLS/src/library/payfine/PayFineUI.java]: All good.

**Review by DChand2021-2023 — COMMENTED** [review]
Great. All defects now addressed.

**Review by DChand2021-2023 — APPROVED** [review]
All done

**Review by RishCSU — APPROVED** [review]
Everything looks dandy! Good to merge :)

**Pull request merged**